client/lib/desktop/index.js

```diff
@@ -117,7 +122,7 @@ var Desktop = {
 
 	onShowMySites: function() {
 		debug( 'Showing my sites' );
-		const site = this.getSelectedSite();
```

Comment:
Huh, that really seems like it should throw an error - the only thing I can think of was some bad copy/paste on my part from the other file in that PR https://github.com/Automattic/wp-calypso/blob/db53a451568ef9040dd4fcb48b8da6140f328709/client/my-sites/sidebar/sidebar.jsx#L109 😞